src/main/resources/application.yaml

```diff
@@ -114,6 +114,10 @@ darts:
     active-directory-b2c-auth-uri: ${darts.azure.active-directory-b2c-base-uri}/hmctsdartsb2csbox.onmicrosoft.com
     active-directory-b2c-external-user-signin-policy: "B2C_1_darts_externaluser_signin"
     active-directory-b2c-external-user-passwordreset-policy: "B2C_1_darts_externaluser_password_reset"
+  daily-list:
+    housekeeping:
+      cron: ${DAILY_LIST_HOUSEKEEPING_SCHEDULER_CRON:4 0/2 * * * MON-FRI}
+      days-to-keep: 30
```

Comment:
I was thinking it might be good to add an "enabled" config in here. If for some reason this job was causing issues it could be disabled with a code change.

Reply:
done